### PR TITLE
test: filter out index files temporarily

### PR DESCRIPTION
**Pull request by KSXGitHub**
Changes in the npm registry cause unnecessary snapshot failures in our tests. The ideal solution is to implement a fake registry that we can control, but until then, filtering them out is good enough.

### CODE REVIEW

**Comment by zkochan** [comment]
How does this mitigate the issue? The snapshot will be invalid if any of the files from the package will change. Or a package will get a new file.

**Comment by KSXGitHub** [comment]
> How does this mitigate the issue?

Create an actual mock registry that we have full control over. Or maybe just a verdaccio proxy is sufficient?

> The snapshot will be invalid if any of the files from the package will change.

Right now, the package versions are pinned. This does mean that we cannot test something involving version ranges (such as `pacquet add foo@latest`), so eventually, we would have to properly solve the problem.

**Comment by zkochan** [comment]
For pnpm we use a verdaccio proxy with prepublished packages for testing: https://github.com/pnpm/registry-mock

I think we should use the same in this repo.

**Review by anonrig — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+	"devDependencies": {
+		"@pnpm/registry-mock": "^3.16.0"
```
Comment (zkochan) [inline package.json:3]: I don't see it being used anywhere. Should it be started by the user?

Comment (KSXGitHub) [reply package.json:3]: I pushed it to wrong branch by mistake.

**Pull request closed without merging**